src/app/games/mancala/ba-awa/BaAwaTutorial.ts

```diff
@@ -0,0 +1,88 @@
+import { Tutorial, TutorialStep } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStep';
+import { MGPOptional } from 'src/app/utils/MGPOptional';
+import { BaAwaConfig } from './BaAwaConfig';
+import { BaAwaRules } from './BaAwaRules';
+import { MancalaTutorial } from '../common/MancalaTutorial';
+import { MancalaState } from '../common/MancalaState';
+import { MancalaDistribution, MancalaMove } from '../common/MancalaMove';
+import { TutorialStepMessage } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStepMessage';
+
+const defaultConfig: MGPOptional<BaAwaConfig> = BaAwaRules.get().getDefaultRulesConfig();
+
+export class BaAwaTutorial extends Tutorial {
+
+    public gameName: string = $localize`Ba-awa`;
+
+    public tutorial: TutorialStep[] = [
+        MancalaTutorial.intro(
+            this.gameName,
+            BaAwaRules.get().getInitialState(defaultConfig),
+        ),
+        TutorialStep.informational(
+            $localize`Ba-awa`,
+            $localize`Bonus fact: Ba-awa, also called Adi, is played mainly in Ghana.`,
+            BaAwaRules.get().getInitialState(defaultConfig),
+        ),
+        MancalaTutorial.sowing(
+            new MancalaState([
+                [4, 4, 4, 4, 4, 4],
+                [4, 0, 4, 4, 4, 4],
+            ], 0, [0, 0]),
+        ),
+        TutorialStep.fromMove(
+            $localize`Multiple laps sowing`,
+            $localize`However, the distribution only stops when the last seed is dropped in a house that contains zero or three seeds (before drop).<br/><br/>You're playing Dark, do such a move!`,
+            new MancalaState([
+                [0, 2, 4, 0, 0, 0],
+                [0, 0, 0, 5, 0, 0],
+            ], 0, [0, 0]),
+            [MancalaMove.of(MancalaDistribution.of(3))],
+            $localize`So, after landing in the house with 2 seeds (then 3), a second lap has been started.`,
+            TutorialStepMessage.FAILED_TRY_AGAIN(),
+        ),
+        TutorialStep.fromMove(
+            $localize`Captures during distribution (1/2)`,
+            $localize`If, during some distribution, you pass by one of your house that contains 3 seeds, and drop a fourth seed, you capture the house immediately, then continue the distribution!<br/><br/>You're playing Dark, do such a move!`,
```

Comment:
ehhh sorry pour les reviews typo :D Mais "one of your houses*"

Reply:
"j'vais être dev moi, j'ai pas besoin d'être bon en dictée madame"